backend/src/main/java/com/woowacourse/kkogkkog/domain/CouponEvent.java

```diff
@@ -53,4 +54,7 @@ private static void canAccept(boolean isSender, boolean isReceiver) {
             throw new ForbiddenException("쿠폰을 보낸 사람만 사용 요청을 수락할 수 있습니다.");
         }
     }
+
+    private static void canFinish(boolean isSender, boolean isReceiver) {
+    }
```

Comment:
제3자가 요청을 보내는 경우에 대한 방어로직이 필요합니다.

```suggestion
    private static void canFinish(boolean isSender, boolean isReceiver) {
        if (!isSender && !isReceiver) {
            throw new ForbiddenException("쿠폰을 보낸 사람과 받은 사람만 사용 완료 처리할 수 있습니다.");
        }
    }
```

Reply:
감사합니다 추가했습니다!